android/modules/ui/src/java/ti/modules/titanium/ui/WebViewProxy.java

```diff
@@ -73,7 +73,6 @@ public WebViewProxy()
 		defaultValues.put(TiC.PROPERTY_OVER_SCROLL_MODE, 0);
 		defaultValues.put(TiC.PROPERTY_LIGHT_TOUCH_ENABLED, true);
 		defaultValues.put(TiC.PROPERTY_ENABLE_JAVASCRIPT_INTERFACE, true);
-		defaultValues.put(TiC.PROPERTY_BORDER_RADIUS, 0);
```

Comment:
I've noticed that this border radius zero line was added to resolve an HTML5 video issue (see PR #8681). Do we need to re-test this? Or have you confirmed that this issue was resolved somewhere else, like in TiUIView?

Reply:
This has already been resolved.